client/components/LearnerViewDirective.js

```diff
@@ -377,8 +376,12 @@ tie.directive('learnerView', [function() {
           question = QuestionDataService.getQuestion(questionId);
           tasks = question.getTasks();
           currentTaskIndex = 0;
+          var storedCode = 
+            CodeStorageService.loadStoredCode(questionId, language);
           $scope.title = question.getTitle();
-          $scope.code = question.getStarterCode(language);
+          $scope.code = 
+            storedCode === null ? 
```

Comment:
Based on linter script, change it to $scope.code = storedCode ? storedCode : question.getStarterCode(language);